### PR TITLE
(QA-3116) add method to return puppetserver.log file

### PR DESCRIPTION
**Pull request by ericldelaney**
Add a method that returns where the puppetserver.log file
so we can check it for specific entries

### CODE REVIEW

File: lib/master_manipulator/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module MasterManipulator
   module Version
-    STRING = '2.0.0'
+    STRING = '2.1.0'
```
Comment (RandellP) [inline lib/master_manipulator/version.rb]: I think, but am not sure.  That the release process may bump this version for you...
https://confluence.puppetlabs.com/display/QA/Releasing+Master+Manipulator

Comment (RandellP) [inline lib/master_manipulator/version.rb]: Just confirmed that it does... was looking in my old PR where Kurt said as much.

Comment (ericldelaney) [inline lib/master_manipulator/version.rb]: ok I remove the version bump

File: lib/master_manipulator/log.rb [hunk 2]
```diff
@@ -2,6 +2,16 @@
 module MasterManipulator
   module Log
 
+    # Return the path to the puppetserver log file
+    # @param [Beaker::Host] master_host The puppet server host
+    # @return [String] path to puppetserver log file
+    # @example return the path to the puppet server log file
+    #   puppet_server_log_path(master)
+    def puppet_server_log_path(master_host)
+      log_dir = on(master_host, "puppet master --configprint logdir").stdout.chomp
```
Comment (johnduarte) [inline lib/master_manipulator/log.rb]: Please use the `puppet` helper method here.

**Comment by ericldelaney** [comment]
@johnduarte updated

**Pull request merged**